webapp-django/crashstats/crashstats/static/crashstats/js/socorro/report.js

```diff
@@ -14,6 +14,12 @@ var SignatureCorrelations = (function() {
                     if (response.count) {
                         $('li.correlations').show();
                     }
+                    if (response.errors) {
```

Comment:
An empty array evaluates to true in JavaScript (try ``if ([]) console.log('yes');`` in your console), so this test should be different. Maybe ``if (response.errors.length) {`` instead?